p2plending/frontend/src/components/componentUtils/getFlag.js

```diff
@@ -1,19 +1,26 @@
 export const getFlagEmoji = lang => {
-    const emojiPath = "https://github.githubassets.com/images/icons/emoji/unicode/";
+    const emojiPath = "https://www.countryflags.io";
+    const emojiStyle = "shiny/32";
+    var langCode = "";
     switch(lang) {
-      case 'English':
-        return `${emojiPath}1f1ec-1f1e7.png`
-      case 'Spanish':
-        return `${emojiPath}1f1ea-1f1f8.png`
-      case 'Chinese':
-        return `${emojiPath}1f1e8-1f1f3.png`
-      case 'Korean':
-        return `${emojiPath}1f1f0-1f1f7.png`
-      case 'Japanese':
-        return `${emojiPath}1f1ef-1f1f5.png`
-      case 'Russian':
-        return `${emojiPath}1f1f7-1f1fa.png?` 
+      case "en":
+        langCode = "us";
+        break;
+      case "ko":
+        langCode = "kp";
+        break;  
+      case "ja":
+        langCode = "jp";
+        break;  
+      case "zh-hanta":
+        langCode = "cn";
+        break;
       default:
-        return 'https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/120/apple/198/waving-white-flag_1f3f3.png';
+        langCode = lang;
     }
+    console.log(`!${lang}!`);
+    console.log(`!${langCode}!`);
+    const flagURl = `${emojiPath}/${langCode}/${emojiStyle}.png`;
+    console.log(flagURl);
```

Comment:
can you remove this console.log? It's generally bad practice to have debugging console logs pushed.

Reply:
will do